internal/pkg/flink/internal/utils/input.go

```diff
@@ -13,25 +13,32 @@ import (
 func GetStdin() *term.State {
 	state, err := term.GetState(int(os.Stdin.Fd()))
 	if err != nil {
-		log.CliLogger.Warnf("Couldn't get stdin state with term.GetState. Error: %v\n", err)
+		log.CliLogger.Warnf("Couldn't get stdin state with term.GetState. Error: %v", err)
 		return nil
 	}
 	return state
 }
 
 func GetConsoleParser() prompt.ConsoleParser {
+	if fileInfo, err := os.Stat("/dev/tty"); err != nil {
+		log.CliLogger.Warnf(`Couldn't open "/dev/tty" file. Error: %v`, err)
+		return nil
+	} else if fileInfo.Mode().Perm()&0444 == 0 {
+		log.CliLogger.Warn(`Couldn't read "/dev/tty" file because read permissions are not set.`)
+		return nil
+	}
 	consoleParser := prompt.NewStandardInputParser()
 	err := consoleParser.Setup()
 	if err != nil {
-		log.CliLogger.Warnf("Couldn't setup console parser. Error: %v\n", err)
+		log.CliLogger.Warnf("Couldn't setup console parser. Error: %v", err)
 	}
 	return consoleParser
 }
 
 func TearDownConsoleParser(consoleParser prompt.ConsoleParser) {
 	err := consoleParser.TearDown()
 	if err != nil {
-		log.CliLogger.Warnf("Couldn't tear down console parser. Error: %v\n", err)
+		log.CliLogger.Warnf("Couldn't tear down console parser. Error: %v", err)
```

Comment:
It seems wrong to be writing "error" in a warning.
```suggestion
		log.CliLogger.Warnf("Couldn't tear down console parser: %v", err)
```